tests/PHPStan/Rules/Operators/data/invalid-binary-mixed.php

```diff
@@ -0,0 +1,157 @@
+<?php declare(strict_types = 1);
+
+namespace InvalidBinaryOperatorWithMixed;
+
+/**
+ * @template T
+ * @param T $a
+ */
+function genericMixed(mixed $a): void
```

Comment:
Some operators are missing from this test:

- `instanceof` is not a binary operator in php-parser.
- `&&`, `||`, `and`, `or` and `xor` - everything can be converted to boolean. And they're already covered by strict-rules.
- `??` is OK to use with mixed.
- Comparison operators are handled by `InvalidComparisonOperationRule`.